src/Acuminator/Acuminator.Analyzers/StaticAnalysis/LegacyBqlField/LegacyBqlFieldAnalyzer.cs

```diff
@@ -25,23 +25,29 @@ internal override void AnalyzeCompilation(CompilationStartAnalysisContext compil
 
 		private async void Analyze(SymbolAnalysisContext context, PXContext pxContext)
 		{
-			if (context.Symbol is INamedTypeSymbol dacFieldType)
+			if (context.Symbol is INamedTypeSymbol dacOrDacExt)
 			{
-				if (!IsDacFieldType(dacFieldType, pxContext) || AlreadyStronglyTyped(dacFieldType, pxContext))
+				if (dacOrDacExt == null || (!dacOrDacExt.IsDAC() && !dacOrDacExt.IsDacExtension()) || context.CancellationToken.IsCancellationRequested)
```

Comment:
There is no need to check for null - the `is` operator performs such checks. `null` value won't pass first `if`

Reply:
done